Library/Homebrew/test/cask/cli/search_spec.rb

```diff
@@ -22,16 +22,38 @@
     EOS
   end
 
+  it "returns matches even when online search failed" do
+    allow(GitHub).to receive(:search_code).and_raise(GitHub::Error.new("reason"))
+    expect {
+      Hbc::CLI::Search.run("local")
+    }.to output(<<-EOS.undent).to_stdout
+      local-caffeine
+      local-transmission
+    EOS
+    .and output(/^Warning: Error searching on GitHub: reason/).to_stderr
+  end
+
   it "shows that there are no Casks matching a search term that did not result in anything" do
     expect {
       Hbc::CLI::Search.run("foo-bar-baz")
-    }.to output("No Cask found for \"foo-bar-baz\".\n").to_stdout.as_tty
+    }.to output(<<-EOS.undent).to_stdout.as_tty
+      No Cask found for "foo-bar-baz".
+    EOS
+  end
+
+  it "doesn't output anything to non-TTY stdout when there are no matches" do
+    expect {
+      Hbc::CLI::Search.run("foo-bar-baz")
+    }.to not_to_output.to_stdout
+    .and not_to_output.to_stderr
   end
 
-  it "lists all available Casks with no search term" do
+  it "lists all Casks available offline with no search term" do
+    allow(GitHub).to receive(:search_code).and_raise(GitHub::Error.new("reason"))
     expect {
       Hbc::CLI::Search.run
     }.to output(/local-caffeine/).to_stdout.as_tty
+    .and not_to_output.to_stderr
```

Comment:
Make the expect a single line and put the `.to` and `.and` line below it.

Reply:
Done in 8e5ad9a. Should I change the rest of the tests to this formatting too?